ci/pipelines/tasks/internal-test-build.yml

```diff
@@ -0,0 +1,21 @@
+inputs:
+- name: source
+
+outputs:
+- name: source
+
+platform: linux
+params:
+    IS_CI: true
+    GOOGLE_PLAY_PRIVATE_KEY: ((apiary-android-secrets.google-play-private-key))
+    KEYSTORE_BASE_64: ((apiary-android-secrets.apiaryMobileJks))
+run:
+    args:
+    - -exc
+    - |
+        cd source
```

Comment:
You can set `dir` to run the script in this location directly instead of `cd`ing to it.